src/components/analyser/search/DashboardCards.tsx

```diff
@@ -0,0 +1,131 @@
+import React from 'react';
+import {
+  Card,
+  CardContent,
+  CardMedia,
+  Grid,
+  Button,
+  Divider,
+  lighten,
+  makeStyles,
+  Theme,
+  Typography,
+  GridList,
+} from '@material-ui/core';
+import classNames from 'classnames';
+import * as API from '../../../analyser/APIClient';
+
+const useStyles = makeStyles((theme: Theme) => ({
+  action: { display: 'inline-block' },
+  row: {
+    cursor: 'pointer',
+  },
+  rowHover: {
+    backgroundColor: lighten(theme.palette.primary.light, 0.85),
+  },
+  defaultText: {
+    fontSize: '24px',
+  },
+  disabled: {
+    cursor: 'not-allowed',
+  },
+  card: {
+    maxWidth: 300,
```

Comment:
maybe add some paddig to the cards, it looks a little bit clusterd

Reply:
done